MultiVersions/Forge/main/beachparty/lang/zh_tw.json

```
@@ -1,125 +1,161 @@
 {
-  "beachparty.config.enableBeachSetBonus": "已啟用沙灘派對套裝獎勵",
-  "beachparty.config.title": "沙灘派對",
+  "advancement.beachparty.build_a_tower": "沙築城堡",
+  "advancement.beachparty.build_a_tower.desc": "建造沙堆城堡──接著讓它更加浩大！",
+  "advancement.beachparty.find_a_overgrown_disc": "復古唱片",
+  "advancement.beachparty.find_a_overgrown_disc.desc": "取得蔓生草枝的唱片",
+  "advancement.beachparty.get_a_seashell": "She Sells Seashells",
+  "advancement.beachparty.get_a_seashell.desc": "收集貝殼",
+  "advancement.beachparty.get_all_beach_items": "碧血金沙灘",
+  "advancement.beachparty.get_all_beach_items.desc": "收集所有種類的目前版本內可取得的飾品",
```

一點吹毛求疵，建議過長的句子，使用逗號換句
或是在不影響原意的情況下，將句子精簡化。
對於玩家來說，模組版本不是他們需要知道的資訊
```suggestion
  "advancement.beachparty.get_all_beach_items.desc": "收集所有種類的飾品",
```